.github/workflows/e2e.yml

```diff
@@ -0,0 +1,29 @@
+name: End-to-end tests
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+    types: [opened, synchronize, ready_for_review]
+
+env:
+  # a list of apps to build and publish on releases
```

Comment:
I don't think you need to specify that actually; only these two apps have the `e2e` target defined, so you can probably just run `npx nx run-many --target=e2e` and it should do the same